app/assets/javascripts/reviews.js

```diff
@@ -1,6 +1,14 @@
 var ready;
 
 ready = function() {
+	$('.myreview-container').parent().css({
+		'box-shadow': 'none',
+		'background-color': '#eee',
+  		'padding-top': '0px',
+  		'padding-left': '0px',
+  		'padding-right': '0px',
+	});
```

Comment:
So, when I change it to overwrite the application.html's #content-container like on the courses page, it overwrites the other html pages of review as well (i.e. create review). That's why I put this in the Javascript, any better way to do this?